cmd/buildah/pull.go

```diff
@@ -78,17 +67,16 @@ func pullCmd(c *cli.Context) error {
 	if len(args) > 1 {
 		return errors.Errorf("too many arguments specified")
 	}
-	if err := parse.ValidateFlags(c, pullFlags); err != nil {
-		return err
-	}
+	// TODO Re-enable
+	//if err := parse.ValidateFlags(c, pullFlags); err != nil {
+	//	return err
+	//}
 
 	systemContext, err := parse.SystemContextFromOptions(c)
 	if err != nil {
 		return errors.Wrapf(err, "error building system context")
 	}
 
-	signaturePolicy := c.String("signature-policy")
```

Comment:
Did you replace this somewhere?

Reply:
yes, because we save the results of the flag in a structure, the assignment is done straight forward.

Reply:
ok